src/Umbraco.Web/PublishedCache/NuCache/DataSource/BTree.DictionaryOfPropertyDataSerializer.cs

```diff
@@ -51,6 +51,11 @@ internal class DictionaryOfPropertyDataSerializer : SerializerBase, ISerializer<
         public void WriteTo(IDictionary<string, PropertyData[]> value, Stream stream)
         {
             // write properties count
+            if (value == null)
```

Comment:
I guess this is a fix for potential bug?

Reply:
Yes. Though it does end up being null for this implementation